src/components/atoms/Button/style.css

```diff
@@ -0,0 +1,46 @@
+.carousel__button {
```

Comment:
carousel styles in button styles

Reply:
Changed names, but I need to think about how to properly name the components.